app/controllers/tasks_controller.rb

```diff
@@ -0,0 +1,53 @@
+class TasksController < ApplicationController
+  def index
+    @tasks = Task.all
+  end
+
+  def show
+    @task = Task.find(params[:id])
+  end
+
+  def new
+    @task = Task.new
+  end
+
+  def create
+    @task = Task.new(task_params)
+    @task.save
+    if @task.save
```

Comment:
If you use `@task.save` on line 16 and 17, it will be saving it twice (unnecessarily)